website/docs/addon_resolve_admin.md

```diff
@@ -15,96 +15,51 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 ## Resolve requirements
-Due to the way resolve handles python and python scripts there are a few steps required steps needed to be done on any machine that will be using AYON with resolve.
+Resolve is using its own python interpreter which cannot be connected to Ayon. This interpreter needs to have installed PySyde2 and OpenTimelineIO dependencies for Ayon to work correctly. Bellow you can find instructions on how to install them into an installed Python of your choice.
 
-### Installing Resolve's own python 3.6 interpreter.
-Resolve uses a hardcoded method to look for the python executable path. All of tho following paths are defined automatically by Python msi installer. We are using Python 3.6.2.
+Resolve 17 is using Python 3.6.2
+Resolve 18 is using Python 3.9.x
 
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
 
-<TabItem value="win">
+### Configuring settings for Python home directory
+To be able to run python in Resolve you need to set `RESOLVE_PYTHON3_HOME` environment variable to the directory path of your Python interpreter version.
 
-`%LOCALAPPDATA%\Programs\Python\Python36`
+Open your server Studio settings and go to **Application** addon `ayon+settings://applications/applications/resolve`. Here you can add the path to the **Environment** section.
 
-</TabItem>
-<TabItem value="linux">
+![Ayon/Application/Resolve](assets/resolve_python_home_application_addon.png)
 
-`/opt/Python/3.6/bin`
 
-</TabItem>
-<TabItem value="mac">
+### Installing Python dependencies
 
-`~/Library/Python/3.6/bin`
+#### PySide2
```

Comment:
On recent Mac I think we'll need to be using PySide6. Right @iLLiCiTiT ?

Reply:
Yeah good point, you are right.